### ISSUE TITLE
error

### ISSUE THREAD

**Issue by energooooo** [TRSoftware1/ChunkBusters#1]
https://pastebin.com/UbzRpwbj
![image](https://user-images.githubusercontent.com/68811273/172074356-b1c27c71-9338-471a-8a6b-04f1f5ac1e07.png)
![image](https://user-images.githubusercontent.com/68811273/172074360-a3818023-5585-48e7-af07-0cc0f0122171.png)

**Comment by TRSoftware1** [comment]
Yes, I've had this issue reported on version 1.16. I haven't noticed it on version 1.18, but I'll be looking into this and applying a fix soon. Thanks for reporting.

**Comment by minehostil** [comment]
Same issue It happens when you click on the item Bust all levels, server version 1.21.3

**Comment by TRSoftware1** [comment]
This should be fixed with recent update. Thank you.
